src/main/java/org/apereo/openlrs/controllers/caliper/CaliperController.java

```diff
@@ -52,7 +58,46 @@
     private CaliperService caliperService;
     @Value("${caliper.version}")
     private String version;
+    @Value("${caliper.pageDefault ?: 0}")
+    private Integer pageDefault;
+    @Value("${caliper.limitDefault ?: 20}")
```

Comment:
The "limit" parameter in the request means "limit the number of events returned in the response to N". The default limit is the number of entries in the response if the request didn't specify a limit. Since a very high limit param in a request could cause server problems (e.g., overload it causing it to crash or ignore other requests), a maximum is imposed to prevent abuse. 

"limit" is the name of the param already used in this application for xAPI features, so I'm reluctant to change it, although I don't think it's the best name, either.  "limit" seems to be the name of the argument to the PageRequest constructor, too. However, I noticed in the response data that it uses the name "number" instead, so I added support for that name as an alias. I may change the names of the variables in the code without changing the API param.